TYPO3.Neos/Classes/TYPO3/Neos/Utility/BackendAssetsUtility.php

```diff
@@ -39,7 +39,7 @@ public function injectSettings(array $settings)
      */
     public function shouldLoadMinifiedJavascript()
     {
-        return isset($this->settings['userInterface']['loadMinifiedJavaScript']) ? $this->settings['userInterface']['loadMinifiedJavaScript'] : $this->settings['userInterface']['loadMinifiedJavascript'];
+        return isset($this->settings['userInterface']['loadMinifiedJavaScript']) ? $this->settings['userInterface']['loadMinifiedJavaScript'] : true;
```

Comment:
WOW, I completely overlooked this and thought it was just a copy/paste bug..
But IMO allowing multiple spellings just leads to more confusion and errors.. I'd rather throw an exception if someone uses the wrong spelling (and/or) if the flag is not set at all.. But that's not the responsibility of the `BackendAssetsUtility` probably